generate-manual.sh

```diff
@@ -2,97 +2,235 @@
+		continue
+	fi
 
-	mkdir -p "$dest_root/$(dirname "$dest")"
+	# create destination directories
+	mkdir -p "$scalable_root/$(dirname "$dest")"
 
+	# copy the icons to the destination
 	if [ -e "../icons/$style/$src.svg" ]; then
-		cp -v "../icons/$style/$src.svg" "$dest_root/$dest.svg"
+		cp -v "../icons/$style/$src.svg" "$scalable_root/$dest.svg"
 	elif [ -e "./icons_linux/$style/$src.svg" ]; then
-		cp -v "./icons_linux/$style/$src.svg" "$dest_root/$dest.svg"
+		cp -v "./icons_linux/$style/$src.svg" "$scalable_root/$dest.svg"
 	else
 		echo "Skipping '$src', icon not found"
 		continue
 	fi
 
-	grep -v 'stroke-width' "$dest_root/$dest.svg" > /dev/null && sed -i 's/\(stroke:[^;]\+\)/\1;stroke-width:1px/g' "$dest_root/$dest.svg"
-	awk -i inplace -F 'stroke-width:|px' "{ print \$1 \"stroke-width:\" (\$2 * $line_weight) \"px\" \$3; }" "$dest_root/$dest.svg"
+	# apply the line weight replacement
+	sed -i ':a;N;$!ba;s/\n//g' "$scalable_root/$dest.svg" # TODO rather fix the regex magic below
+	grep -v 'stroke-width' "$scalable_root/$dest.svg" >/dev/null && sed -i 's/\(stroke:[^;]\+\)/\1;stroke-width:1px/g' "$scalable_root/$dest.svg"
+	awk -i inplace -F 'stroke-width:|px' "{ print \$1 \"stroke-width:\" (\$2 * $line_weight) \"px\" \$3; }" "$scalable_root/$dest.svg"
 
+	# create symbolic links for duplicate icons
 	if [ ${#dests[@]} -gt 1 ]; then
-		for i in $(seq 1 $((${#dests[@]}-1))); do
-			mkdir -p "$dest_root/$(dirname "${dests[$i]}")"
-			ln -vs "../${dests[0]}.svg" "$dest_root/${dests[$i]}.svg"
+		for i in $(seq 1 $((${#dests[@]} - 1))); do
+			mkdir -p "$scalable_root/$(dirname "${dests[$i]}")"
+			rm "$scalable_root/${dests[$i]}.svg" 2>/dev/null || true
+			ln -vsr "$scalable_root/$dest.svg" "$scalable_root/${dests[$i]}.svg"
 		done
 	fi
-done
 
-if type inkscape; then
-	for line in $(cat mapping.txt); do
-		echo $line
-		src=$(echo "$line" | cut -d, -f1)
-		dests=( `echo "$line" | cut -d, -f2 | tr ':' ' '` )
-		dest=${dests[0]}
-		dest_root="./arcticons/symbolic"
-		src_root="./arcticons/scalable"
+	# create symbolic icons
+	if $inkscape; then
+		mkdir -p "$symbolic_root/$(dirname "$dest")"
+		echo "creating symbolic icon '$symbolic_root/$dest-symbolic.svg'"
+		inkscape \
+			--actions="select-all;object-stroke-to-path" \
+			--export-filename="$symbolic_root/$dest-symbolic.svg" \
+			--export-overwrite \
+			"$scalable_root/$dest.svg" || true
+		# shellcheck disable=SC2015
+		scour --quiet "$symbolic_root/$dest-symbolic.svg" "$symbolic_root/$dest-symbolic.tmp.svg" &&
+			rm "$symbolic_root/$dest-symbolic.svg" &&
+			mv "$symbolic_root/$dest-symbolic.tmp.svg" "$symbolic_root/$dest-symbolic.svg" || true
 
-		mkdir -p "$dest_root/$(dirname "$dest")"
-		inkscape --actions="select-all;object-stroke-to-path" --export-filename="$dest_root/$dest-symbolic.svg" "$src_root/$dest.svg" || true
-		rm $src_root/$dest*.0.svg || true
+		# why is this line here?
+		rm "$scalable_root/$dest*.0.svg" 2>/dev/null || true
```

Comment:
This line exists because in GTK environments, Inkscape crashes because of a bug in GTK3, generating files ending with `.0.svg`. I think a better solution would be simply to put `**/*.0.svg` into the `.gitignore`